be/src/exec/vectorized/chunks_sorter_topn.cpp

```diff
@@ -499,6 +499,11 @@ Status ChunksSorterTopn::_hybrid_sort_common(RuntimeState* state, std::pair<Perm
         // take sort_row_number rows from permutations.second merge-sort with _merged_segment.
         _merge_sort_common(big_chunk, segments, sort_row_number, sorted_size, permutation_size, new_permutation.second);
 
+        if (big_chunk->reach_capacity_limit()) {
```

Comment:
can we use an unified macro to check capacity limit alike CHECK_MEMORY_LIMIT ?

Reply:
That's too trivial, cause every place need difference error message, and maybe got difference type of chunk.